apps/Gruntfile.js

```diff
@@ -67,6 +67,35 @@ module.exports = function (grunt) {
   var ace_suffix = envOptions.dev ? '' : '-min';
   var dotMinIfNotDev = envOptions.dev ? '' : '.min';
   var piskelRoot = String(child_process.execSync('`npm bin`/piskel-root')).replace(/\s+$/g,'');
+  var PISKEL_DEVELOPMENT_MODE = grunt.option('piskel-dev');
+  if (PISKEL_DEVELOPMENT_MODE) {
+    var localNodeModulesRoot = String(child_process.execSync('npm prefix')).replace(/\s+$/g,'');
+    if (piskelRoot.indexOf(localNodeModulesRoot) === -1) {
+      // Piskel has been linked to a local development repo, we're good to go.
+      piskelRoot = path.resolve(piskelRoot, '..', 'dev');
+      console.log(chalk.bold.yellow('-- PISKEL DEVELOPMENT MODE --'));
+      console.log(chalk.yellow('Make sure you have a local development build of piskel'));
+      console.log(chalk.yellow('Inlining PISKEL_DEVELOPMENT_MODE=true'));
+      console.log(chalk.yellow('Copying development build of Piskel instead of release build'));
+
+    } else {
+      console.log(chalk.bold.red('Unable to enable Piskel development mode.'));
+      console.log(chalk.red('In order to use Piskel development mode, your apps ' +
+          'package must be linked to a local development copy of the Piskel ' +
+          'repository with a complete dev build.' +
+          '\n' +
+          '\n  1. git clone https://github.com/code-dot-org/piskel.git <new-directory>' +
+          '\n  2. cd <new-directory>' +
+          '\n  3. npm install && grunt build-dev' +
+          '\n  4. npm link' +
+          '\n  5. cd -' +
+          '\n  6. npm link @code-dot-org/piskel' +
+          '\n  7. rerun your previous command' +
+          '\n'));
```

Comment:
I believe our minimum supported (and suggested) node version [is currently v0.12.4](https://github.com/code-dot-org/code-dot-org/blame/staging/SETUP.md#L81) (although our package.json says [we support back to 0.10.25](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/package.json#L7) I think that's a lie).  0.12 [does not support template literals](https://kangax.github.io/compat-table/es6/#node012) or much ES6 at all, really.